.travis.yml

```diff
@@ -1,6 +1,10 @@
 # R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
 
 language: R
+r:
+  - 3.4
```

Comment:
I would prefer to change the travis yaml independent of the other changes.

Reply:
makes sense! I was trying to debug #363. Will revert.